src/main/java/core/machines/Truck.java

```diff
@@ -0,0 +1,14 @@
+package core.machines;
+
+public class Truck extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Truck started work");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Truck stopped working");
```

Comment:
Similarly, the message in the `stopWork` method should be more informative and indicate that the `Truck` has stopped doing its specific work, for example, 'Truck has finished hauling cargo'.